### ISSUE TITLE
Requiring published at date and having a checkbox for published status doesn't make sense

### ISSUE THREAD

**Issue by illegalnumbers** [jipiboily/monologue#299]
When I'm creating a blog post I think it doesn't make much sense to have a published at date as well as a checkbox for whether or not it's published. Especially if the published at date is a required date, that was a little annoying. I think it would make more sense to autogenerate that field on initial publication date by clicking the checkmark stating that the draft has been published.

**Comment by msevestre** [comment]
This is used for future post.

**Comment by illegalnumbers** [comment]
Right, but it's required for any post to be created. It's also not obvious that the publishing happens automatically or the time of the publishing (iirc...could be wrong on that) that will occur. It might be a better situation to opt in to that  behavior rather than having it by default.
